docs/index.rst

```diff
@@ -1,408 +1,66 @@
-=====================
-urllib3 Documentation
-=====================
+urllib3
+=======
 
 .. toctree::
    :hidden:
+   :maxdepth: 2
 
-   pools
-   managers
-   security
-   helpers
-   exceptions
-   collections
-   contrib
-   recipes
+   user-guide
+   advanced-usage
+   reference/index
+   contributing
 
+Urllib3 is a *sanity-friendly* HTTP client for Python. It helps you avoid the
+inevitable heartache and frustration of lower-level HTTP clients. It is the
+magic beneath the fantastic `Requests <http://python-requests.org/>`_ library.
```

Comment:
Will do. No need to deep-dive yet, as this is more of a "sanity check".

Thanks.